cmd/srcd/cmd/parse_test.go

```diff
@@ -0,0 +1,271 @@
+// +build integration
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	cmdtest "github.com/src-d/engine/cmd/test-utils"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ParseTestSuite struct {
+	cmdtest.IntegrationSuite
+	testDir string
+}
+
+func TestParseTestSuite(t *testing.T) {
+	s := ParseTestSuite{}
+	suite.Run(t, &s)
+}
+
+type testCase struct {
+	url      string
+	filename string
+	lang     string
+}
+
+// Uses files from https://github.com/leachim6/hello-world
+var testCases = []testCase{
```

Comment:
Isn't it better to have these files locally in a fixture dir?

Reply:
yes please :pray:

Reply:
It's funny how obvious it sounds now that you mention it.

Done in 990247d

Reply:
😂

Reply:
LGTM 👍, thank you